zerver/openapi/zulip.yaml

```diff
@@ -3259,10 +3259,6 @@ paths:
                                       description: |
                                         Whether this organization is configured to allow users to access
                                         [message edit history](/help/view-a-messages-edit-history).
-                                    allow_message_deleting:
-                                      type: boolean
-                                      description: |
-                                        Whether messages can be deleted in this Zulip organization.
```

Comment:
We can remove `message_content_delete_limit_seconds` also from here, but was not sure how to add those fields in the `realm/update` schema.

Reply:
On second thought, I think It would be better if all those unncessary fields are removed in a single commit making it a single entry in changelog. I can do this but I am not sure how to add these fields and their description in the current schema of `realm/update` after removing these from here. (related issue - #18697)